server/api/jurisdictions.py

```diff
@@ -12,23 +12,25 @@
 
 
 def serialize_jurisdiction(
-    jurisdiction: Jurisdiction, round_status: Optional[JSONDict]
+    election: Election, jurisdiction: Jurisdiction, round_status: Optional[JSONDict]
 ) -> JSONDict:
-    return {
+    json_jurisdiction = {
```

Comment:
should there be a simple status field that indicates the type of audit? Or is that assumed to be loaded from the election metadata?

Reply:
it's loaded in the election settings already

Reply:
👍